test/integration/targets/ec2_group/vars/main.yml

```diff
@@ -0,0 +1 @@
+#has_ec2_classic: "{{ lookup('aws_account_attribute', attribute='has-ec2-classic') }}"
```

Comment:
Did you mean to keep this file here?

Reply:
No, that's really weird.  The git rm for this is in my shell history but ah well! Thx.